toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1014,18 +1061,19 @@ PreferencesPopup::PreferencesPopup()
   }
   styleSheetType->addItems(styleSheetList);
   styleSheetType->setCurrentIndex(currentIndex);
-
+  //m_pixelsOnlyCB->setChecked(m_pref->getPixelsOnly());
+  m_pixelsOnlyCB->setChecked(true);
```

Comment:
Please insert the following in order to disable the combo box if the "pixel" unit is selected on launch.
```
if (m_pref->getPixelsOnly()){
  m_unitOm->setDisabled(true);
  m_cameraUnitOm->setDisabled(true);
 }
```